ZenPacks/zenoss/Microsoft/Windows/WinService.py

```diff
@@ -98,7 +98,7 @@ def monitored(self):
 
         # 2 - Check what our template says to do.
         datasource = self.getMonitoredDataSource()
-        if datasource and self.getMonitored(datasource):
+        if datasource and self.getMonitored(datasource) and datasource.enabled:
```

Comment:
Does this change also need to be made to "getMonitoredDataSource" (line 179) and "setMonitoredDataSource" (line 191)?